class Test1
	has Foo  # .error. Cannot find
	
	pass


class Test2
	has foo  # .error. Cannot find
	
	pass
